gdx/src/com/badlogic/gdx/utils/StreamUtils.java

```diff
@@ -119,9 +119,11 @@ public static String copyStreamToString (InputStream input, int estimatedSize, S
 
 	/** Close and ignore all errors. */
 	public static void closeQuietly (Closeable c) {
-		if (c != null) try {
-			c.close();
-		} catch (Exception ignored) {
+		if (c != null) {
+			try {
+				c.close();
+			} catch (Exception ignored) {
+			}
 		}

```

Comment:
I believe this is now done.  Merged, rebased and force-pushed to my forked branch.